### PR TITLE
Fixes #1870

### LINKED ISSUE #1870: Linux Build Instructions should clone from https
**Affects versions :**

Building from GitHub clone

**Describe the bug**

Instructions should state to clone from https, not http.

**Additional context**

https://github.com/xournalpp/xournalpp/blame/master/readme/LinuxBuild.md#L60

### PR DESCRIPTION
**Pull request by rickhg12hs**
Fixes #1870

### CODE REVIEW

File: readme/LinuxBuild.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ common available, therefore the Cairo PDF Export is without PDF Bookmarks.
 
 The binary executable will be placed in the `build/src/` subdirectory.
 
+To place the binary executable and support files in systemwide directories, execute `sudo make install`.
+
+After updating xournalpp, with perhaps `git pull`, execute `cmake ..`, `cmake --build .`, and optionally `sudo make install` again.
+
```
Comment (Technius) [inline readme/LinuxBuild.md:79]: These are steps that we explicitly discourage below.

Comment (rickhg12hs) [reply readme/LinuxBuild.md:79]: I did read that, but I don't understand why.  The procedure I mention works well for me.  Why shouldn't I do it?

Comment (Technius) [reply readme/LinuxBuild.md:79]: There is no clean way to uninstall after running `sudo make install`. If you upgrade or uninstall xournalpp, you risk leaving behind files that could cause problems down the road. Additionally, `sudo make install` is likely to be incompatible with your system's package manager.

There is a `rpm/fedora/xournalpp.spec` file if you would like to build an RPM package.

Comment (rickhg12hs) [reply readme/LinuxBuild.md]: OK, I made some changes.

File: readme/LinuxBuild.md [hunk 2]
```diff
@@ -73,6 +73,8 @@ common available, therefore the Cairo PDF Export is without PDF Bookmarks.
 
 The binary executable will be placed in the `build/src/` subdirectory.
 
+After updating xournalpp, with perhaps `git pull`, execute `cmake ..` and `cmake --build .`.
```
Comment (rickhg12hs) [inline readme/LinuxBuild.md:76]: Acceptable?

Comment (Febbe) [reply readme/LinuxBuild.md:76]: You should also mention to cd into the builddir. Also an "touch ../CMakeLists.txt" should be called before "cmake .."

**Pull request closed without merging**